### PR TITLE
net-news/newsboat: add 2.33, add doc USE flag

### PR DESCRIPTION
**Pull request by zlamas**
Closes: https://bugs.gentoo.org/831123
Closes: https://bugs.gentoo.org/910210

### CODE REVIEW

File: net-news/newsboat/newsboat-2.33.ebuild [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+src_compile() {
+	export CARGO_HOME="${ECARGO_HOME}"
+	emake prefix="/usr" CXX="$(tc-getCXX)" AR="$(tc-getAR)" RANLIB="$(tc-getRANLIB)" \
+		newsboat podboat mo-files $(usev doc)
+}
+
+src_test() {
+	emake CXX="$(tc-getCXX)" AR="$(tc-getAR)" RANLIB="$(tc-getRANLIB)" check || die
```
Comment (mm1ke) [inline net-news/newsboat/newsboat-2.33.ebuild:182]: no need to add that `|| die` here

**Comment by M1cR0xf7** [comment]
i lost hope this ebuild will every get updated. i will compile it myself

**Comment by thesamesam** [comment]
> i lost hope this ebuild will every get updated. i will compile it myself

Not a particularly helpful comment. You should be able to maintain it in a local repository as well.

**Comment by mortezadadgar** [comment]
This has been fixed https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=02ed578c12e9e4e109a62eaa8561018eb77a9b82

**Pull request closed without merging**